mirgecom/euler.py

```diff
@@ -76,31 +77,36 @@
 class ConservedVars:  # FIXME: Name?
     r"""Resolve the canonical conserved quantities.
 
-    Get the canonical conserved quantities (mass, energy, momentum)
-    per unit volume = $(\rho,\rho{E},\rho\vec{V})$ from an agglomerated
-    object array.
+    Get the canonical conserved quantities (mass, energy, momentum,
+    mass_fraction) per unit volume = $(\rho,\rho{E},\rho\vec{V},
+    \rho{Y_s})$ from an agglomerated object array.
 
-    .. attribute:: dim
+    .. attribute:: dim: int
 
-    .. attribute:: mass
+    .. attribute:: mass: numpy.ndarray
 
         Mass per unit volume
 
-    .. attribute:: energy
+    .. attribute:: energy: numpy.ndarray
 
         Energy per unit volume
 
-    .. attribute:: momentum
+    .. attribute:: momentum: numpy.ndarray
 
         Momentum vector per unit volume
```

Comment:
Doc shape (and fix how the type is documented)

Reply:
This changed over several commits.  Can you have another look here, please?
https://github.com/illinois-ceesd/mirgecom/blob/55b37f304d597f27e139d032b9a3ed53006f23e9/mirgecom/euler.py#L109